.travis.yml

```diff
@@ -1,25 +1,26 @@
-language: rust
+language: minimal
```

Comment:
I wasn't able to install node in a way so that the existing installation is no longer present. That is why I switched to the `minimal` image that doesn't have any language-bias.